scripts/xor-codec/hash_codec.py

```diff
@@ -0,0 +1,91 @@
+import os
+import sys
+import numpy
+import hashlib
+
+valid_hashes = [
+	# PyTorch Checkpoint V1:
+	"f856e9d99c30855d6ead4d00cc3a5573",
+	"d9dbfbea61309dc1e087f5081e98331a",
+	"2b2bed47912ceb828c0a37aac4b99073",
+	"ea0405cdb5bc638fee12de614f729ebc",
+	"4babdbd05b8923226a9e9622492054b6",
+	# PyTorch Checkpoint V2:
+	"d9dbfbea61309dc1e087f5081e98331a",
+	"f856e9d99c30855d6ead4d00cc3a5573",
+	"4babdbd05b8923226a9e9622492054b6",
```

Comment:
Is there any reason to include these duplicate hashes (they are all the same as Checkpoint V1)?

Reply:
It's fine to eliminate. The entire hash list could be deduplicated without breaking the program.